rpc/lib/test/main.go

```diff
@@ -28,10 +28,13 @@ func main() {
 	cdc := amino.NewCodec()
 	logger := log.NewTMLogger(log.NewSyncWriter(os.Stdout))
 	rpcserver.RegisterRPCFuncs(mux, routes, cdc, logger)
-	_, err := rpcserver.StartHTTPServer("0.0.0.0:8008", mux, logger, rpcserver.Config{})
+	listener, err := rpcserver.Listen("0.0.0.0:8008", rpcserver.Config{})
 	if err != nil {
 		cmn.Exit(err.Error())
 	}
+	if err := rpcserver.StartHTTPServer(listener, mux, logger); err != nil {
```

Comment:
goroutine?

Reply:
Nope, how could the caller possibly block execution otherwise? IMHO this has to block hereand return error. Ditto for the one above

Reply:
My bad, I didn't see it. Fixed though